client/js/components/index.js

```diff
@@ -3,5 +3,7 @@ import Grid from './grid'
 import Input from './Input'
 import Note from './Note/Note'
 import NoteTab from './NoteTab/notesTabs'
+import Message from './Message'
+import styles from './Layout/Layout.styles'
 
-export {Button, Grid, Input, Note, NoteTab}
+export {Button, Grid, Input, Note, NoteTab, Message, styles}
```

Comment:
Perhaps we should move/rename the `styles` export to something more descriptive like `siteStyles` or `layoutStyles`.